### PR TITLE
Edit/test rule generator

### PR DESCRIPTION
**Pull request by seongjinyoon**
## Description
- Fixed wrong test assertions that had previously existed due to a bug from the previous `mo-sql-parsing` library version regarding `ILIKE`.
- Test cases 2 and 9 fixed.

### CODE REVIEW

File: server/wsgi.py
Comment (baiqiushi) [inline server/wsgi.py:1]: Why do we need to change this file in this PR?

**Review by baiqiushi — CHANGES_REQUESTED** [review]
Let's remove unnecesary changes.

File: server/wsgi.py
Comment (seongjinyoon) [reply server/wsgi.py:1]: My mistake. Resolved.

**Review by baiqiushi — APPROVED** [review]
LGTM.

**Pull request merged**